Server/Plugins/APIDump/Classes/Plugins.lua

```diff
@@ -22,10 +22,10 @@ return
 
 	cPluginLua =
 	{
-		Desc = "",
+		Desc = "(<b>OBSOLETE</b>) This class is no longer useful in the API and will be removed as soon as all core plugins are migrated away from it. The {{cPlugin}} class serves as the main plugin instance's interface.",
```

Comment:
And I guess I got mine too.

:)

-----

How did you solve the problem of the web tab implementation leaks, which was your concern when I proposed to remove cPluginLua?